listenbrainz/webserver/static/js/src/playlists/Playlists.tsx

```diff
@@ -164,7 +167,7 @@ export default class UserPlaylists extends React.Component<
         "Duplicated playlist",
         <>
           Duplicated to playlist&ensp;
-          <a href={`/playlist/${newPlaylistId}`}>{newPlaylistId}</a>
+          <a href={`/playlist/${newPlaylistId}`}>Copy of {playlistTitle}</a>
```

Comment:
is this name generated by the frontend when it's duplicated, or by the backend? If it's by the backend it should probably use a name returned by the backend, in the case that we decide to change this name.

Reply:
It requires fetching the newly created playlist (since the API endpoint only returns the new playlist MBID), but it is probably wiser.